.github/workflows/main.yml

```diff
@@ -357,6 +357,52 @@ jobs:
         with:
           name: optimization${{ matrix.python-version }}
           path: ./o${{ matrix.python-version }}/*
+  Tutorials-Stable:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+    if: ${{ startsWith(github.ref, 'refs/heads/stable') }}
```

Comment:
Why only on the stable branch? To provide coverage to prevent headaches that result in things like #1178 we should run this on master to ensure that the tutorials run on the proposed next 0.x.0 release (which is HEAD of master). That way if a breaking change is proposed we will catch that and either fix it, or update the tutorials well ahead of a release.

Reply:
I agree with you. We already have a failure detected on Aqua master. I had done just for stable but already changed my mind. I will talk to @woodsp-ibm to confirm it.